src/components/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Since you're using react 18 version, there is no need to import `react` here. React does it for you.